index.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>University Portal</title>
+    <link rel="stylesheet" href="./assets/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;1,100&display=swap" rel="stylesheet">
+</head>
+
+<body id="body">
+    <header>
+        <nav class="navbar" id="navbar">
+
+            <div class="navbarMenuClose"><i class="close" id="close"><img src="./assets/images/icons/icon-cancel.png"
+                        alt="icon-cancel"></i>
+            </div>
+            <div class="firstRow">
+                <ul>
+                    <li><i><img class="social" src="./assets/images/icons/linkedin.svg" alt="linkedin"></i></li>
+                    <li><i><img class="social" src="./assets/images/icons/twitter.svg" alt="twitter"></i></li>
+                    <li>English</li>
+                    <li>My Page</li>
+                    <li>Logout</li>
+                </ul>
+            </div>
+            <div class="secondRow">
+                <div class="logo"><a href="./index.html"> Global University</a></div>
+
+                <ul class="navbarLinks">
+                    <li class=""><a href="./about.html">About</a></li>
+                    <li class=""><a href="#">Program</a></li>
+                    <li class=""><a href="#">Join</a></li>
+                    <li class=""><a href="#">Sponsor</a></li>
+                    <li class=""><a href="#">News</a></li>
+                    <li class=""><a href="#">cc campaign</a></li>
+                </ul>
+            </div>
+
+        </nav>
+    </header>
+    <main id="main">
+        <section class="intro">
+            <div class="blur">
+                <img src="./assets/images/hamburger.svg" alt="hamburger" class="hamburger" id="humbarger">
+                <span class="welcome">Well come to Global University</span>
+                <h1 class="title">Global University is the most famous University in the Europa</h1>
+                <div class="introDescription">
+                    This year Global University accepts more than 3 thousand of students from out side of the Europa
+                </div>
+                <div class="introFooter">2023 New admission is in progress</div>
+            </div>
+        </section>
+        <section class="programs">
+            <h1 class="mainProgram">Main Programs</h1>
```

Comment:
- [ ] While you did a fantastic job following the semantics for HTML, it is required to use only one `h1` tag per page, because creates hierarchy and gives your page a more logical structure. It makes it easy for readers (and those using screen readers) to skim and understand what your page is about :+1:, so kindly make sure that you only use one `<h1>` for each page